src/server/index.js

```diff
@@ -110,6 +110,16 @@ app.get('/csta18', (req,res) => renderView(res, 'csta18.pug'));
 app.get('/cybersecurity', (req,res) => renderView(res, 'roboscape.pug'));
 app.get('/roboscape', (req,res) => res.redirect('/cybersecurity'));
 
+// 2019 camp files
+app.get('/camp2019/netsblox', (req,res) => renderView(res, 'camp2019/netsblox.pug'));
```

Comment:
@hamidzr - where are these files? They don't seem to be included in the PR (I missed this when first merging it :/ )...

Reply:
maybe we went with `/roboscaspe` instea and thats why this wasn't finished :thinking: